Sources/HummingbirdPostgres/Migration.swift

```diff
@@ -18,7 +18,7 @@ import PostgresNIO
 /// Protocol for a database migration
 ///
 /// Requires two functions one to apply the database migration and one to revert it.
-public protocol PostgresMigration {
+public protocol PostgresMigration: Sendable {
```

Comment:
Is there a reason this has Sendable conformance?

Reply:
The validate script added the Sendable conformance.

Reply:
> Is there a reason this has Sendable conformance?

Same as the internal init removal.

Reply:
oh that's odd. Is that SwiftFormat trying to be overly clever? What version of SwiftFormat do you have?

Reply:
> oh that's odd. Is that SwiftFormat trying to be overly clever? What version of SwiftFormat do you have?

```sh swiftformat --version
0.54.3
```
It looks like version 0.54.3